config/template.py

```diff
@@ -0,0 +1,50 @@
+import os
+import binascii
+
+def render_template(template, destination, **kwargs):
+
+
+   here = os.path.abspath(os.path.dirname(__file__))
+   template = os.path.join(here, 'kinto.tpl') 
+   destination = os.path.join(here, 'kinto.ini')
+
+    with open(template, 'r') as f:
```

Comment:
We should probably use ``codecs.open`` to handle UTF-8 in this files.

See: https://github.com/mozilla-services/cliquet/blob/master/setup.py#L8-L9